phpseclib/Crypt/Base.php

```diff
@@ -2001,11 +2057,48 @@ function _createInlineCryptFunction($cipher_code)
      * for which $mode the lambda function was created.
      *
      * @access private
-     * @return &Array
+     * @return Array &$functions
      */
     function &_getLambdaFunctions()
     {
         static $functions = array();
         return $functions;
     }
+
+    /**
+     * Generates a digest from $bytes
+     *
+     * @see _setupInlineCrypt()
+     * @access private
+     * @param $bytes
+     * @return String
+     */
+    function _trapdoor($bytes)
+    {
+        if (!defined('CRYPT_BASE_WHIRLPOOL_AVAILABLE')) {
```

Comment:
I don't think they are. I mean, PHP4 supports static variables in methods but not in classes. And static variables in methods can't be seen outside of those methods.

Example: http://3v4l.org/6R7cr#v430

PHP5+ gives no output. PHP4.x gives a parse error.